src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-param-reassign */
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      state.books.push(action.payload);
+    },
+    removeBook: (state, action) => {
+      state.books = state.books.filter((book) => book.id !== action.payload);
+    },
+  },
+});
+
+export const { addBook } = booksSlice.actions;
```

Comment:
Well done for creating the actions for adding and removing books. You seemed to miss out on adding the removeBook action in your export. Kindly include it.